importer/Uploader.py

```diff
@@ -246,17 +245,18 @@ def upload(self):
     def set_wd_item(self):
         item_exists = True if self.data["wd-item"] is not None else False
         if item_exists:
-            # item_q = self.data["wd-item"]
-            self.wd_item = self.wdstuff.QtoItemPage(self.TEST_ITEM)
-            print(self.wd_item)
+            item_q = self.data["wd-item"]
```

Comment:
I have addressed this now; just using the --upload argument on its own will use the Sandbox mode, in order to go `live` you actually have to add that as an argument. That way you have to explicitly request the live mode, which prevents triggering it by accident. I have also updated the README.